compiler/rustc_target/src/spec/targets/armv7_rtems_eabihf.rs

```diff
@@ -0,0 +1,39 @@
+use crate::spec::{cvs, Cc, LinkerFlavor, Lld, PanicStrategy, RelocModel, Target, TargetOptions};
+
+pub fn target() -> Target {
+    Target {
+        llvm_target: "armv7a-unknown-none-eabihf".into(),
+        metadata: crate::spec::TargetMetadata {
+            description: None,
+            tier: Some(3),
+            host_tools: Some(false),
+            std: Some(true),
+        },
+        pointer_width: 32,
+        data_layout: "e-m:e-p:32:32-Fi8-i64:64-v128:64:128-a:0:32-n32-S64".into(),
+        arch: "arm".into(),
+
+        options: TargetOptions {
+            os: "rtems".into(),
+            families: cvs!["unix"],
+            abi: "eabihf".into(),
+            linker_flavor: LinkerFlavor::Gnu(Cc::Yes, Lld::No),
+            linker: Some("arm-rtems6-gcc".into()),
```

Comment:
Ideally I would like to remove this line and add the actual name of the linker later. I am currently working on a `rtems-sys` crate to add the RTEMS-specific configuration to an application project.

Do you have any tips how to achieve that?

Reply:
If linkers always have the version number (no `arm-rtems-gcc` and regular `arm-...-gcc` doesn't work) then it is probably better to just leave it as `None`. It can always be specified with `-Clinker` when invoking `rustc` (https://doc.rust-lang.org/rustc/codegen-options/index.html#linker) or by setting the Cargo config https://doc.rust-lang.org/cargo/reference/config.html?highlight=linker#targettriplelinker.